src/OpenDiffix.Core.Tests/Value.Tests.fs

```diff
@@ -44,3 +44,21 @@ module ComparerTests =
     [ Integer 10L; Null; Integer 1L; Null; Integer 5L ]
     |> sort Descending NullsLast
     |> should equal [ Integer 10L; Integer 5L; Integer 1L; Null; Null ]
+
+  [<Fact>]
+  let ``String sort groups by letter case`` () =
+    [ String "a"; String "C"; String "A"; String "c" ]
+    |> sort Ascending NullsLast
+    |> should equal [ String "a"; String "A"; String "c"; String "C" ]
```

Comment:
I am confused by this test (which I guess should belong in the previous commit) and the previous commit title.
The sort should be case insensitive, so the expected order should be `aACc`, but it is not.

Reply:
My bad, the tests ended up committed once, for the bulk change, I was doing a little bit of back and forth here.

On the expected order, I think we need the sort to be in the order, where the case is a secondary sort criteria. So not precisely "case insensitive" as I described the commit. Case insensitive would mean `aACc` indeed in this case, I made a mental shortcut, not thinking about it's not the same thing.

This way or another, the test is in line with what PostgreSQL does and what we should be doing.